unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Tests/AvatarSlotComponentViewShould.cs

```diff
@@ -121,5 +121,59 @@ public void SetTooltipText2()
             Assert.AreEqual(avatarSlot.tooltipCategoryText.text, $"{TEST_CATEGORY}");
             Assert.False(avatarSlot.tooltipHiddenText.gameObject.activeInHierarchy);
         }
+
+        [Test]
+        public void SetHiddenBy()
+        {
+            avatarSlot.model = new AvatarSlotComponentModel()
```

Comment:
We should be using a method like `avatarSlot.SetModel(...)` which internally does the `RefreshControl()` like the rest of the UI components. Goes for the rest of the tests aswell